base_field_validator/models/ir_model.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    See __openerp__.py about license
+#
+##############################################################################
+
+import re
+from openerp.tools.translate import _
+from openerp import SUPERUSER_ID
+
+from openerp import models, fields, api, _
+from openerp.exceptions import Warning
+from openerp.osv import orm
+
+
+class IrModel(models.Model):
+
+    _inherit = 'ir.model'
+    validator_line_ids = fields.One2many(
+        'ir.model.validator.line', 'model_id', 'Validators')
```

Comment:
I'm just wondering if this should go linked in `ir.model.fields` instead of `ir.model`. What do you think?